configuration/NI.Analysis.targets

```diff
@@ -0,0 +1,110 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- TODO: This location of 'content' needs to get updated once we build our package if we decide to put the files elsewhere, outside of a 'content' directory -->
+    <CodeAnalysisRuleSetDirectory>$(PkgNI_DotNet_Analyzers)\content</CodeAnalysisRuleSetDirectory>
+    <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+  </PropertyGroup>
+
+	<PropertyGroup>
```

Comment:
(Optional) Perhaps we should put a condition on this e.g. `<PropertyGroup Condition="'$(IsNITestProject)' == ''">`. That way, if a project is being classified incorrectly, it can fix itself by manually setting `<IsNITestProject>true/false</IsNITestProject>`. Then, I'd add another PropertyGroup for `IsNITestUtilitiesProject`.

Reply:
I like this idea. I'm going to do this in a follow-up submission where we have a good diff on this one. Adding to backlog.